### PR TITLE
Companion PR for Substrate#4394

### PR DESCRIPTION
**Pull request by expenses**
See https://github.com/paritytech/substrate/pull/4394.

### CODE REVIEW

File: cli/src/lib.rs [hunk 1]
```diff
@@ -27,6 +27,7 @@ use chain_spec::ChainSpec;
 use futures::{
 	Future, FutureExt, TryFutureExt, future::select, channel::oneshot, compat::Future01CompatExt,
 };
+#[cfg(not(target_os = "unknown"))]
```
Comment (tomaka) [inline cli/src/lib.rs]: In Substrate, this code is in a `cli` module that is behind a `cli` feature:
https://github.com/paritytech/substrate/blob/6e0a4877afbe2309e106b84333bb70200531b604/bin/node/cli/src/lib.rs#L38

What do you think of doing the same here?

**Comment by arkpar** [comment]
https://github.com/paritytech/polkadot/pull/723/commits/d7a305f1fe1e1edccbbe3a974066ad91a702dc58 Should be reverted I guess

File: cli/src/lib.rs (hunk 1 above)
Comment (expenses) [reply cli/src/lib.rs]: Done

**Comment by expenses** [comment]
Either this is #737 can be merged to get these browser-utils changes.

**Pull request merged**